### PR TITLE
NGDOC-183: add chinese translation

### PR DESCRIPTION
**Pull request by fangchen-li**

### CODE REVIEW

**Review by burtonquantil — COMMENTED** [review]
why are you modifying the English version of the file?

**Comment by fangchen-li** [comment]
> why are you modifying the English version of the file?

The changes in the English version appear to be the result of Markdown syntax errors and are not consistent with the previous ones.

File: zh/docs/edge-logic/faq.md [hunk 1]
```diff
@@ -126,4 +126,48 @@ proxy_ignore_headers Vary;
...
+CDN Pro 使用指令 [`origin_fast_route`](</docs/edge-logic/supported-directives.md#origin_fast_route>) 来进行回源时与源站之间的加速。 这个强大的功能基于我们屡获殊荣的 [High-speed Data Transmission](https://www.cdnetworks.com/enterprise-applications/high-speed-data-transmission/) (HDT) 技术。它确保了 CDN Pro 的服务器使用最优的回源链路，即使在某些极端恶劣的网络链路情况下也能保证服务的稳定性。此指令亦可用于某些源站链路不佳，但是首次 MISS 请求性能又极其重要的可缓存业务上。通过 [`origin_fast_route`](</docs/edge-logic/supported-directives.md#origin_fast_route>) 服务的流量会因其带来额外成本而收取更高的费用。要试用此功能，请联系网宿（CDNetworks）技术支持。
+
+### CDN Pro API呼叫率如何受到限制?
+
+为了防止CDN Pro的API服务器过载，CDN Pro API强制限制客户每分钟可以发送的请求数量。如果客户发送过多的请求，API速率限制通过返回带有HTTP状态码429的错误消息来限制客户。
+
+* **令牌桶算法**
+
+CDN Pro采用令牌桶算法实现速率限制。令牌桶算法基于一个固定容量的桶的类比，令牌以固定的速度添加到桶中。在允许一个API请求继续之前，要检查桶，看它是否至少包含一个令牌。如果是，则从桶中删除一个令牌，允许API请求继续进行。如果可用的令牌数量不足，则会阻止请求，并出现一个错误，提示在1分钟的滑动窗口内已超过配额。
+桶的容量和填充速率由客户管理API中`configs.apiMaxBurst`和`configs.apiRate`字段控制。例如，如果客户有`configs.apiMaxBurst = 45`和`configs.apiRate = 120`，则以每分钟120个令牌的固定速度向桶中添加令牌，桶的总容量为45。水桶以一种“贪婪”的方式被填满。CDN Pro试图尽快向桶中添加令牌。例如，每分钟120个token即每500毫秒增加1个token。换句话说，添加不会等待整整一分钟来再生整堆120个token。
```
Comment (AlexQuantil) [inline zh/docs/edge-logic/faq.md:138]: Some places you used '令牌' while others used ‘token’
The translation needs to be consistent.

File: zh/docs/edge-logic/faq.md [hunk 2]
```diff
@@ -126,4 +126,48 @@ proxy_ignore_headers Vary;
...
+
+* **指示灯和错误处理**
```
Comment (AlexQuantil) [inline zh/docs/edge-logic/faq.md:140]: 错误提示

File: zh/docs/edge-logic/faq.md [hunk 3]
```diff
@@ -126,4 +126,48 @@ proxy_ignore_headers Vary;
...
+
+在令牌被消费之后，一个`X-rate-limit-remaining: X`报头被添加到API调用的HTTP响应中，表示桶中剩余的令牌数量。此报头表示客户此时可以发出的API请求的剩余配额。失败或格式错误的请求也会从桶中消耗一个令牌。如果桶中没有足够数量的令牌，API网关返回一个HTTP 429错误，响应头为`x-rate-limit-retry-after-seconds: Y`，告诉客户端在Y秒后重试。
```
Comment (AlexQuantil) [inline zh/docs/edge-logic/faq.md:142]: some used '报头' while others used '响应头'

File: zh/docs/edge-logic/faq.md [hunk 4]
```diff
@@ -126,4 +126,48 @@ proxy_ignore_headers Vary;
...
+
+* **避免速率限制错误的最佳实践**
```
Comment (AlexQuantil) [inline zh/docs/edge-logic/faq.md:144]: 调用频率限制 not 速率限制

File: zh/docs/edge-logic/faq.md [hunk 5]
```diff
@@ -126,4 +126,48 @@ proxy_ignore_headers Vary;
...
+
+检查API请求历史记录。`GET /ngadmin/apicalls` 显示了你调用的API。(调用此API需要客户管理API凭据。)仔细检查记录，看是否有滥用的可能。
```
Comment (AlexQuantil) [inline zh/docs/edge-logic/faq.md:146]: 客户管理员账号，not 客户管理API凭据

File: zh/docs/edge-logic/faq.md [hunk 6]
```diff
@@ -126,4 +126,48 @@ proxy_ignore_headers Vary;
...
+通过使用更适合该场景的API或将类似的请求合并为一个请求来减少请求的数量。例如，如果您想监控一个域列表的流量，一种无效的方法是:
+
+对于domain_list中的domain:
```
Comment (AlexQuantil) [inline zh/docs/edge-logic/faq.md:149]: ？？？？？Did you really check the machine translation?
```
for domain: domain_list:
...
```
This is a code block.

File: zh/docs/edge-logic/faq.md [hunk 7]
```diff
@@ -126,4 +126,48 @@ proxy_ignore_headers Vary;
...
+调用 `GET /cdn/report/vol`
+`{filters: {hostnames: [$domain]}}`
+
+相反，请使用以下推荐的方法:
+
+`POST /cdn/report/volSummary`
+`{filters: {hostnames: [$domain_list]}, groupBy: [hostnames]}`
+
+另一个例子是在一个请求中清除多个文件，而不是为每个清除URL调用API。如果有数千个清除URL遵循某种模式，则在创建清除请求时通过指定dirUrls或regexppatterns字段来使用目录或正则表达式清除。请求的数量不应该随着您获得更多的域、属性和其他资源而增加。
```
Comment (AlexQuantil) [inline zh/docs/edge-logic/faq.md:158]: purge: 刷新
please check the word reference: https://github.com/mileweb/ngportal-language

File: zh/docs/edge-logic/faq.md [hunk 8]
```diff
@@ -126,4 +126,48 @@ proxy_ignore_headers Vary;
...
+
+如果通过脚本调用API，它应该对间歇性或非特定错误具有弹性。脚本应该执行`x-rate-limit-retry-after-seconds`报头，并在延迟后重试请求。考虑在您的代码中包含一个进程，它可以调节请求的速率，以便它们能均匀地随时间分布。
```
Comment (AlexQuantil) [inline zh/docs/edge-logic/faq.md:160]: be resilient to：
能够处理和应对
not 具有弹性

File: zh/docs/edge-logic/faq.md [hunk 9]
```diff
@@ -126,4 +126,48 @@ proxy_ignore_headers Vary;
...
+
+如果问题仍然存在，请联系我们的技术支持团队。如果有合理的需要增加速率限制或突发上限，技术支持团队将评估您的需求并提高阈值。
+
+
+* **注意**
+
+速率限制适用于客户级别。同一个客户下的所有API帐户共享同一个令牌桶。过度使用一个帐户将耗尽客户的配额。转售商的子客户的速率限制是相互独立的。此外，子客户的API调用不会使用父客户的配额。
+
+因为CDN Pro门户调用API，所以门户UI中的操作也使用API速率限制令牌。
```
Comment (AlexQuantil) [inline zh/docs/edge-logic/faq.md:169]: Portal不建议翻译成门户。
因为Portal UI也会调用API
即可

**Comment by AlexQuantil** [comment]
> why are you modifying the English version of the file?
Hi Burton, I saw other chapters are all using
`* ** subtitle **`
instead of
`** subtitle **`
image
So I suggested Fangchen change it to make them consistent.

**Comment by AlexQuantil** [comment]
@burtonquantil BTW, the chapter [How are CDN Pro API calls rate limited](https://docs.quantil.com/en/cdn/docs/edge-logic/faq#how-are-cdn-pro-api-calls-rate-limited) is under "Edge Logic Reference - FAQ". Shouldn't we put it into "API Reference - FAQ" since it's nothing related to the edge logic?

**Comment by burtonquantil** [comment]
@AlexQuantil  @nibinqtl  there is only one FAQ now, so I placed it in the existing one.
An API FAQ with just one question?
By the way, I did chat with Ele about adding an icon on the home page to refer to it, so it's more accessible like Getting Started, Release Notes, etc.  Right now, it's a bit hidden.

**Comment by burtonquantil** [comment]
> > why are you modifying the English version of the file?
> > Hi Burton, I saw other chapters are all using
> > `* ** subtitle **`
> > instead of
> > `** subtitle **`
> > image
> > So I suggested Fangchen change it to make them consistent.

OK.
I'm not sure who made the earlier FAQs with the indented heading. I thought that looked odd, so I did it differently for API rate limiting, but making them consistent is fine. 
Thanks

**Comment by AlexQuantil** [comment]
> @AlexQuantil @nibinqtl there is only one FAQ now, so I placed it in the existing one. An API FAQ with just one question? By the way, I did chat with Ele about adding an icon on the home page to refer to it, so it's more accessible like Getting Started, Release Notes, etc. Right now, it's a bit hidden.

We can create a topic under "Authentication" or somewhere. Not necessarily in the FAQ.
image

**Comment by AlexQuantil** [comment]
@fangchen-li I've updated the doc. Please double-check the content and polish the sentence.

**Pull request merged**